test/e2e/conformance_test.go

```diff
@@ -67,6 +68,14 @@ var _ = Describe("Conformance Tests", func() {
 	Measure(specName, func(b Benchmarker) {
 		var err error
 
+		kubernetesVersion := e2eConfig.GetVariable(capi_e2e.KubernetesVersion)
+		flavor := clusterctl.DefaultFlavor
+		if useCIArtifacts {
+			flavor = "conformance-ci-artifacts"
+			kubernetesVersion, err = kubernetesversions.LatestCIRelease()
+			Expect(os.Setenv("CI_VERSION", kubernetesVersion)).NotTo(HaveOccurred())
```

Comment:
In the case where you call a func that returns only an error, you can rewrite it to be a bit more readable:

`Expect(os.Setenv("CI_VERSION", kubernetesVersion)).To(Succeed())`